operators/src/processing/reprojection.rs

```diff
@@ -251,6 +251,7 @@ impl RasterOperator for Reprojection {
             measurement: in_desc.measurement.clone(),
             time: in_desc.time,
             bbox,
+            resolution: None, // TODO: calculate resolution
```

Comment:
I think it is very problematic because the resolution can highly vary, depending on the location. We also have this problem when determining the resolution with which we actually query the source in the reprojection. 

I think a first approach would be to build special methods for known conversions which are very important to us. E.g. 4326<->3857 or 4326<->UTM32N. I can't think of any generic solutions that would work between arbitrary projections... Maybe reprojecting the bounds of every pixel and taking the minimum width and height, but that would be very expensive.

Reply:
Johannes said something about an algorithm from GDAL…